tests/test_json.py

```diff
@@ -2,8 +2,7 @@
 
 from useq import MDASequence
 
-JSON = '{"metadata": {"some info": "something"}, "axis_order": "tpcz", "stage_positions": [{"x": 10.0, "y": 20.0, "z": null, "name": null, "z_plan": {"go_up": true}}, {"x": 10.0, "y": 20.0, "z": 50.0, "name": "test_name", "z_plan": {"go_up": true, "above": 10.0, "below": 0.0, "step": 1.0}}], "channels": [{"config": "Cy5", "group": "Channel", "exposure": 50.0, "do_stack": true, "z_offset": 0.0, "acquire_every": 1, "camera": null}, {"config": "FITC", "group": "Channel", "exposure": 100.0, "do_stack": true, "z_offset": 0.0, "acquire_every": 1, "camera": null}, {"config": "DAPI", "group": "Channel", "exposure": null, "do_stack": false, "z_offset": 0.0, "acquire_every": 3, "camera": null}], "time_plan": {"prioritize_duration": false, "phases": [{"prioritize_duration": false, "interval": 3.0, "loops": 3}, {"prioritize_duration": true, "interval": 10.0, "duration": 2400.0}]}, "z_plan": {"go_up": true, "range": 1.0, "step": 0.5}}'  # noqa
-
+JSON = '{"metadata": {"some info": "something"}, "axis_order": "tpcz", "stage_positions": [{"x": 10.0, "y": 20.0, "z": null, "name": null, "z_plan": {"go_up": true}, "tile_plan": {"overlap": 0.0, "snake_order": true}}, {"x": 10.0, "y": 20.0, "z": 50.0, "name": "test_name", "z_plan": {"go_up": true, "above": 10.0, "below": 0.0, "step": 1.0}, "tile_plan": {"overlap": 0.0, "snake_order": true}}], "tile_plan": {"overlap": 0.0, "snake_order": true, "rows": 2, "cols": 1, "relative_to": "center"}, "channels": [{"config": "Cy5", "group": "Channel", "exposure": 50.0, "do_stack": true, "z_offset": 0.0, "acquire_every": 1, "camera": null}, {"config": "FITC", "group": "Channel", "exposure": 100.0, "do_stack": true, "z_offset": 0.0, "acquire_every": 1, "camera": null}, {"config": "DAPI", "group": "Channel", "exposure": null, "do_stack": false, "z_offset": 0.0, "acquire_every": 3, "camera": null}], "time_plan": {"prioritize_duration": false, "phases": [{"prioritize_duration": false, "interval": 3.0, "loops": 3}, {"prioritize_duration": true, "interval": 10.0, "duration": 2400.0}]}, "z_plan": {"go_up": true, "range": 1.0, "step": 0.5}}'  # noqa
```

Comment:
let's put this into a new nicely formatted .json file (and load it as a fixture) ... makes it much better for git to see what has changed (I should have done that in the first place)

Reply:
actually never mind, I'll do that in another pr

Reply:
ok, you can merge in main now